src/journal/bsod.c

```diff
@@ -194,13 +215,27 @@ static int display_emergency_message_fullscreen(const char *message) {
                 goto cleanup;
         }
 
+        if (message_id) {
+                r = loop_write(fd, "\nScan the QR code below, and search for the entry with the ID: ", SIZE_MAX);
+                if (r < 0)
+                        return log_warning_errno(r, "Failed to write emergency message to terminal: %m");
+
+                r = loop_write(fd, message_id, SIZE_MAX);
```

Comment:
Maybe prefix this with a short string "Message ID: " or so, that should be enough